docs/cudf/source/cudf_pandas/usage.md

```diff
@@ -18,20 +24,31 @@ df.groupby("size").total_bill.mean()  # uses the GPU
 df.apply(list, axis=1)                # uses the CPU (fallback)
 ```
 
-## Command-line usage
+## Command Line Usage
 
 From the command line, run your Python scripts with `-m cudf.pandas`:
 
 ```bash
 python -m cudf.pandas script.py
 ```
 
-### Usage in tandem with
-[`multiprocessing`](https://docs.python.org/3/library/multiprocessing.html)
-or
-[`concurrent.futures`](https://docs.python.org/3/library/concurrent.futures.html)
-process pools
+This will make `import pandas` use `cudf.pandas` acceleration with zero code change.
+
+## Import Usage
+
+Another way to enable `cudf.pandas` is via explicit import. If you are running
+a script where you cannot control the command line flags, this is another
+option that enables `cudf.pandas` with minimal code changes. Be sure that
+`cudf.pandas.install()` is called _before_ importing `pandas`.
```

Comment:
Related to the above, should we explicitly state that this mode also works in Jupyter notebooks?